src/main/java/org/jenkinsci/plugins/GithubAuthorizationStrategy.java

```diff
@@ -216,6 +216,32 @@ public boolean isAllowAnonymousJobStatusPermission() {
         return rootACL.isAllowAnonymousJobStatusPermission();
     }
 
+    /**
+     * Compare an object against this instance for equivalence.
+     * @param object An object to campare this instance to.
+     * @return true if the objects are the same instance and configuration.
+     */
+    @Override
+    public boolean equals(Object object){
+        if(object instanceof GithubAuthorizationStrategy) {
+            GithubAuthorizationStrategy obj = (GithubAuthorizationStrategy) object;
+            return (
+                    this.getOrganizationNames().equals(obj.getOrganizationNames()) &&
+                    this.getAdminUserNames().equals(obj.getAdminUserNames()) &&
+                    this.isUseRepositoryPermissions() == obj.isUseRepositoryPermissions() &&
+                    this.isAuthenticatedUserCreateJobPermission() == obj.isAuthenticatedUserCreateJobPermission() &&
+                    this.isAuthenticatedUserReadPermission() == obj.isAuthenticatedUserReadPermission() &&
+                    this.isAllowGithubWebHookPermission() == obj.isAllowGithubWebHookPermission() &&
+                    this.isAllowCcTrayPermission() == obj.isAllowCcTrayPermission() &&
+                    this.isAllowAnonymousReadPermission() == obj.isAllowAnonymousReadPermission() &&
+                    this.isAllowAnonymousJobStatusPermission() == obj.isAllowAnonymousJobStatusPermission()
+                    );
+        }
+        else {
```

Comment:
Unusual coding style with one brace on the same line as the `else` -- I'm not familiar with this plugin's style though.